src/sentry/profiles/consumers/process/factory.py

```diff
@@ -20,37 +21,29 @@ def process_message(message: Message[KafkaPayload]) -> None:
     if sampled or options.get("profiling.profile_metrics.unsampled_profiles.enabled"):
         b64encoded_uncompressed = b64encode(message.payload.value).decode("utf-8")
 
-        if random.random() < options.get("taskworker.try_compress.profile_metrics.rollout"):
-            import time
-            import zlib
-
-            metrics.distribution(
-                "profiling.profile_metrics.uncompressed_bytes", len(b64encoded_uncompressed)
-            )
+        metrics.distribution(
+            "profiling.profile_metrics.uncompressed_bytes", len(b64encoded_uncompressed)
+        )
```

Comment:
Double base64 encoding just to create a metric seems wasteful. We could base the metric on the `message.payload.value` instead :shrug: